presto-native-execution/CMakeLists.txt

```diff
@@ -142,7 +142,8 @@ include_directories(SYSTEM ${FOLLY_INCLUDE_DIRS})
 
 # Include third party header files
 
-find_package(Protobuf REQUIRED)
```

Comment:
We need to enable this and check the version first before depending on the Velox bundled.

Reply:
However, cmake will fail if Protobuf is not found. Even if a preinstalled protobuf is found, if the version of the preinstalled protobuf is not compatible with the version required by Velox, we can not build the codebase successfully. I was blocked here for a long time.

Reply:
@bianhq I will take a look this week. Sorry for the delay on this.